### PR TITLE
Services

### PR DESCRIPTION
**Pull request by courageousillumination**
This is mostly infrastructure stuff right now. No actual server is started with this code, but we can now create services and connect them together.

### CODE REVIEW

File: server/deckr/services/service_starter.py [hunk 1]
```diff
@@ -55,7 +36,8 @@ def add_service(self, service_config, config_for_service):
             config_for_service (dict): Configuration to pass to the service upon creation.
         """
 
-        pass
+        wrapper = ServiceWrapper(service_config, config_for_service)
+        self.services[service_config["name"]] = wrapper
 
     def start(self):
```
Comment (alexthebake) [inline server/deckr/services/service_starter.py]: This function could probably be broken up into smaller functions. It's may not that necessary, but every comment in this function is essentially a function name.

File: server/deckr/services/service_wrapper.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+This module provides the code for a simple service wrapper.
+"""
+
+import importlib
+import logging
+
+LOGGER = logging.getLogger(__name__)
+
+
+class ServiceWrapper(object):
```
Comment (alexthebake) [inline server/deckr/services/service_wrapper.py]: I like this class.

**Comment by alexthebake** [comment]
This looks great to me. I'm gonna go ahead and merge the request.

File: server/deckr/services/service_starter.py (hunk 1 above)
Comment (courageousillumination) [inline server/deckr/services/service_starter.py]: Yeah cood point. I'll break it up in the next round of code.

**Pull request merged**